packaging/standalone/src/main/distribution/shell-scripts/bin/neo4j

```diff
@@ -215,38 +213,38 @@ startit() {
 
     echo -n "process [$STARTED_PID]"
 
-	  if [ $WAIT = "true" ] ; then
-	    echo -n "... waiting for server to be ready."
-	    while kill -0 $STARTED_PID 2> /dev/null ; do
-	      ## wait for start, pick up the server listening on the port
-	      detectrunning
-	      if [ $newpid ] ; then
-	         break
```

Comment:
Pretty please try to avoid this amount of formatting changes in scripts, because as it's hard to read changes through simultaneous formatting change in Java code (which we are familiar with), it's even harder for scripts which feels much more uncomfortable making changes in.